annotator/static/app.js

```diff
@@ -12,5 +12,7 @@ $(() => {
         $container: $("#player"),
         videoSrc: window.video.location,
         videoId: window.video.id,
+        videoStart: window.video.start_time,
```

Comment:
The video object gives us actually very limited useful initial state. It will give us a total of two variables:

1. location
2. start time

It does not store the following info:

1. end time
2. id
3. anything else